internal/host/hostcommands/empty_cmd.go

```diff
@@ -0,0 +1,24 @@
+package hostcommands
+
+import (
```

Comment:
Why not use this one https://github.com/openshift/assisted-service/blob/master/internal/host/hostcommands/noop_cmd.go ?

Reply:
noop_cmd does not suit the purpose, it does not return any steps to be sent to the agent. W need a step sent to agent, but that its command is empty

Reply:
This cause an error since we are trying to run empty command.  Why is there a need to send a step to the agent that does nothing ?

Reply:
we want agent to re-register. we also don't want to change agent's code. So we send it an empty command ( since we don't want it to do anything), and tell it to exit when it is done. also the response in the AI does not affect host state

Reply:
So use the noop command above.  It will do the same, but without the error

Reply:
fixed